### ISSUE TITLE
Clarification for Fire Hydrants

### ISSUE THREAD

**Issue by vdubya** [csi-net/OmniClass#2]
Need clarification on the difference between the following:
* 22-21 11 16, Facility Fire Hydrants
* 22-33 12 19, Water Utility Distribution Fire Hydrants

Version: Table 22, 2013-08-25

**Comment by csi-net** [comment]
Table 22 stuff can usually be clarified by reference to MasterFormat. In this case, the associated informational content reads as follows:

21 11 00 Facility Fire-Suppression Water-Service Piping
Includes: piping systems for fire suppression for a structure.
Fire hydrants closely associated with a structure.
May Include: fire suppression water service from the structure
to the utility water service line.
See: 21 30 00 for fire pumps.
33 14 16 for site water utility distribution piping
33 14 19 for valves and hydrants for water utility service

21 11 16 Facility Fire Hydrants
21 11 19 Fire-Department Connections

Division 33 got a major re-work in the 2016 edition that has not yet spread to OmniClass. Going back to the MasterFormat 2012 edition for this OmniClass version (most recent balloted through NBIMS):

33 12 00 Water Utility Distribution Equipment
Includes: water distribution equipment for a facility site and
for multiple facilities.
See: 21 11 16 for facility fire hydrants.
26 42 00 for cathodic protection.
33 11 00 for water utility distribution piping.
33 12 13 Water Service Connections
33 12 13.13 Water Supply Backflow Preventer Assemblies
33 12 16 Water Utility Distribution Valves
33 12 19 Water Utility Distribution Fire Hydrants
33 12 23 Water Utility Pumping Stations
33 12 33 Water Utility Metering

In short, water utility distribution fire hydrants is campus wide service to hydrants. Actual hydrants close to buildings or piping in or near buildings is 22-21 11 16.

Admittedly the title for Water Utility Distribution Fire Hydrants could have been less clumsy.

I hope that helps.

Greg Ceton, CSI, CDT
Director of Strategic Initiatives and Special Projects
CSI
110 South Union Street, Suite 100
Alexandria, VA 22314
v: 703-706-4723 | f: 703-236-6047
gceton@csinet.org<mailto:gceton@csinet.org> | Skype: gceton | Twitter: @gceton<https://twitter.com/gceton> | LinkedIn<http://www.linkedin.com/in/gceton>
CSI Basecamp Site<https://constructionspecificationsinstitute.basecamphq.com/>

**Comment by vdubya** [comment]
Very helpful, thanks!
